docs/src/Components.jsx

```diff
@@ -210,6 +210,21 @@ var Components = React.createClass({
           </ul>
         </article>
 
+        <article id="weather">
+          <h3>Weather</h3>
+          <p>Display the weather for a current day in a standard format. Combine a few to create a forcast i.e 5-day</p>
```

Comment:
I don't think this reads quite right, should it 'be for a day' instead of 'for a current day'?